lemur/plugins/lemur_aws/elb.py

```diff
@@ -421,11 +422,67 @@ def attach_certificate_v2(listener_arn, port, certificates, **kwargs):
     :param certificates:
     """
     try:
-        return kwargs["client"].modify_listener(
+        needs_cert_update_for_sni = has_listener_cert_for_sni(listener_arn, kwargs["client"])
+
+        modified_listener = kwargs["client"].modify_listener(
             ListenerArn=listener_arn, Port=port, Certificates=certificates
         )
+
+        if needs_cert_update_for_sni:
```

Comment:
Can we add a test case for this, or do we already have a case internally that we can verify post-deployment?

Reply:
Though seemed straightforward, [describe_listener_certificates](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/elbv2/client/describe_listener_certificates.html) output for mocked client is missing the `IsDefault` flag which should've been present as per the docs. Implementation depends on the flag being present.
The code is tested manually rotating cert for an LB. And we do have a case to internally verify post-deployment